services/src/main/java/org/keycloak/protocol/oidc/endpoints/LoginStatusIframeEndpoint.java

```diff
@@ -51,7 +55,20 @@ public LoginStatusIframeEndpoint(KeycloakSession session) {
     @GET
     @Produces(MediaType.TEXT_HTML_UTF_8)
     public Response getLoginStatusIframe(@QueryParam("version") String version) {
-        return returnIframeFromResources("login-status-iframe.html", version, session);
+        final var map = new HashMap<String, Object>();
+        final var isSecureContext = SecureContextResolver.isSecureContext(session);
+        final var adminBaseUri = session.getContext().getUri(UrlType.ADMIN).getBaseUri();
```

Comment:
I wonder if we should be using a different endpoint here. What happens if the admin console is [exposed on a different hostname](https://www.keycloak.org/server/hostname#_exposing_the_administration_console_on_a_separate_hostname) that is not publicly available? Perhaps this should be `UrlType.FRONTEND`? Note I am not fully aware of the implications of this due to lack of JavaDoc on this enum.